data.json

```
@@ -0,0 +1,899 @@
+{
+  "assessedAt": "2025-04-22T11:00:00Z",
+  "assessedBy": {
+    "id": "mailto:mlieberman85@gmail.com",
+    "comment": "Automated scan + manual review based on internal checklist #789"
+  },
+  "assessmentComment": "Full Self-Assessment for OpenSSF Security Baseline for WTF Frontend",
+  "controls": [
+    {
+      "id": "OSPS-AC-01.01",
+      "conclusion": "non-compliant",
+      "evidence": [
+        {
+          "humanEvidence": {
+            "name": "manual-inspection",
+            "humanDescription": "Examined AuthN MFA requirements for the GitHub Organization and found lack of enforcement. Changing the setting could be disruptive to the organization and will require further planning to mitigate."
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-AC-02.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
+      ]
+    },
+    {
+      "id": "OSPS-AC-03.01",
+      "conclusion": "compliant",
+      "evidence": [
+        {
+          "ellaEvidence": {
+            "name": "automated-scan",
+            "ella": {
+              "uri": "",
+              "uriData": {}
+            }
+          }
+        }
```

Here's a good place for an example of what automated attestation looks like.  Note that `uriData` does not include fields like "id", "name", "_links", or "created_at" in this case.

```suggestion
        {
          "ella": {
            "uri": "https://github.com/repos/wtf/frontend/rulesets/5171149",
            "uriData": {
              "bypass_actors": [],
              "conditions": {
                "ref_name": {
                  "exclude": [],
                  "include": ["~DEFAULT_BRANCH"]
                }
              },
              "enforcement": "active",
              "rules": [
                {"type": "deletion"},
                {"type": "non_fast_forward"},
                {"type": "pull_request"}
              ]
            }
          }
        }
```